crates/bevy_ui/src/render/mod.rs

```diff
@@ -456,17 +455,27 @@ pub fn extract_uinode_borders(
                 Option<&TargetCamera>,
                 Option<&Parent>,
                 &Style,
-                &BorderColor,
+                Option<&BorderColor>,
+                Option<&Outline>,
```

Comment:
I think this would be a nice use-case for `AnyOf` :)

Reply:
Ah that's good this bit was annoying me actually, I didn't know we had an `AnyOf` filter now.